pennylane/devices/tests/test_gradients_tf.py

```diff
@@ -152,6 +152,10 @@ def test_multi_meas(self, diff_method, device, tol):
         wires = 3 if diff_method == "hadamard" else 2
         dev = device(wires=wires)
         tol = tol(dev.shots)
+
+        if "lightning" in getattr(dev, "name", "").lower():
+            pytest.skip("tf interfaces not working correctly with lightning")
```

Comment:
Wait, so we just skip tests if the device is TF and uses Lightning? Any way for us to fix this?
If not, should we instead of skipping mark it as xfail?

Reply:
We can mark it as xfail, but since the device test suite doesn't run with lightning devices on the PL repo, once this issue is fixed it will raise errors in lightning CI and block work again.

Reply:
So this is a stopgap for now? Can we annotate this area though with a TODO so we can track work needs to be done here, and create a SC story for it to be rectified in future?

Reply:
Added TODOs, and opened [this issue](https://github.com/PennyLaneAI/pennylane/issues/5519) to track this.

Reply:
Basically, tensorflow dtypes are always a massive headache and timesink.